src/lib/fcitx/instance_p.h

```diff
@@ -133,6 +133,8 @@ class InstancePrivate : public QPtrHolder<Instance> {
     watchEvent(EventType type, EventWatcherPhase phase, EventHandler callback);
 
 #ifdef ENABLE_KEYBOARD
+    static std::optional<std::string> getLocale();
```

Comment:
I think it's better to just return std::string, as the previous `std::optional` is because `getEnvironment` returns so. Of course, usages in `InstancePrivate::InstancePrivate` needs to be updated.